host/sgx/sgxquote.c

```diff
@@ -135,6 +136,191 @@ static quote3_error_t (*_sgx_qv_verify_quote)(
 static void* _ql_module;
 static void* _qvl_module;
 
+// This is a helper for getting "human readable error messages" - as mentioned
+// in issue 2945 - so that we do not have to return complex quote3_error_t codes
+// in error messages.
+static const char* get_quote3_error_t_string(quote3_error_t err)
+{
+    static char quote3_error_t_number[16];
+    sprintf_s(
+        quote3_error_t_number, sizeof(quote3_error_t_number), "0x%x", err);
```

Comment:
Since quote3_error_t_number is only used in the default case if I understand right, move this down into the default case

Reply:
Make sense. Updated.